packages/phone-number-privacy/signer/src/signing/get-partial-signature.ts

```diff
@@ -43,6 +46,11 @@ export async function handleGetBlindedMessagePartialSig(
   logger.info({ request: request.body }, 'Request received')
   logger.debug('Begin handleGetBlindedMessagePartialSig')
 
+  const key: Key = {
+    name: KeyName.phoneNumberPrivacy,
+    version: Number(request.headers.keyVersion) || config.keystore.keys.phoneNumberPrivacy.latest,
```

Comment:
Same suggestion as above.
```suggestion
    version: Number(request.headers['Odis-Key-Version'] ?? config.keystore.keys.phoneNumberPrivacy.latest),
```

Reply:
I think this might be one of the few places where || is appropriate over ??. My reasoning is the Number() could return NaN which is falsy. We also don't support version '0' of keys. I'm basing this off the following stackoverflow post. Let me know if that checks out.  https://stackoverflow.com/questions/23437476/in-typescript-how-to-check-if-a-string-is-numeric/23440948#23440948